### PR TITLE
Added feature revert to the database update cloud hook

### PR DESCRIPTION
**Pull request by pfrilling**
## Summary
This adds a custom db-update hook to run database updates and features-import for the eCMS project when the site is changed to a new codebase.

## Metadata
| Question | Answer |
|----------|--------|
| Did you use a meaningful pull request title? | yes
| Did you apply meaningful labels to the pull request? | yes
| Did you perform a self review first? | yes
| Documentation reflects changes? | no
| `CHANGELOG` reflects changes? | yes
| Unit/Functional tests reflect changes? | n/a
| Did you perform browser testing? | yes
| Risk level | Medium
| Relevant links | [RIG-96](https://thinkoomph.jira.com/browse/rig-96)

### CODE REVIEW

File: factory-hooks/db-update/site-update.sh [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+# custom_argument="$5"
+
+# The websites' document root can be derived from the site/env:
+docroot="/var/www/html/$site.$env/docroot"
+
+# Acquia recommends the following two practices:
+# 1. Hardcode the drush version.
+# 2. When running drush, provide the application + url, rather than relying
+#    on aliases. This can prevent some hard to trace problems.
+DRUSH_CMD="drush8 --root=$docroot --uri=https://$domain"
```
Comment (ben-hamelin) [inline factory-hooks/db-update/site-update.sh:34]: Checking drush version - I think we might need drush 9 or drush 10?

**Review by ben-hamelin — COMMENTED** [review]
Looks good, just a check on the drush version. Locally my composer.lock is on Drush 10.3.4

File: factory-hooks/db-update/site-update.sh (hunk 1 above)
Comment (pfrilling) [reply factory-hooks/db-update/site-update.sh]: I changed the version to drush 10. I also confirmed this is available on the Acquia servers.

**Comment by ben-hamelin** [comment]
Ok looks good!
/approved

**Pull request merged**